### ISSUE TITLE
New Device: Zigbee switch suppoet

### ISSUE THREAD

**Issue by tasict** [iRayanKhan/homebridge-tuya#120]
**Describe your device**
Tuya Zigbee switch and zigbee gateway 

**Partial Support?**
Does this device work in any fashion in the plugin? 
no

**Additional context**
no

**Comment by bkarlson** [comment]
Yes, it is relevant. Basically Tuya BLE or ZigBee require a hub/gateway:
https://www.aliexpress.com/wholesale?SearchText=tuya+gateway&opensearch=true
https://expo.tuya.com/product/707006

Currently this plugin cannot see devices exposed via such gateway. 

It is a real bummer, because many sensors require a gateway and can be exposed via Tuya Web BUT Tuya web APi does not support push events, i.e. motions sensors are pretty much useless this way. 

I can offer necessarly logs or whatever to help development in this direction.
